src/cache/client.py

```
@@ -128,16 +125,8 @@ async def is_deriver_flush_enabled() -> bool:
-        # Strip query parameters - redis-py doesn't support custom params like ?suppress=true
-        parsed = urlparse(settings.CACHE.URL)
-        clean_url = urlunparse(parsed._replace(query=""))
-
-        redis_client = aioredis.from_url(clean_url)  # pyright: ignore[reportUnknownMemberType]
-        try:
-            result = await redis_client.get(DERIVER_FLUSH_KEY)
-            return result == b"1"
-        finally:
-            await redis_client.aclose()
+        result = await cache.get(DERIVER_FLUSH_KEY)
+        return result in ("1", b"1", 1)
```

Ruff (0.14.14)

[warning] Consider moving this statement to an `else` block

(TRY300)